schunk_description/urdf/lwa4p_extended/lwa4p_extended_without_base.urdf.xacro

```diff
@@ -15,7 +15,7 @@
       <axis xyz="0 0 1"/>
       <calibration rising="${arm_1_ref}"/>
       <dynamics damping="0" friction="0"/>
-      <limit effort="81.5" velocity="0.42" lower="-4.712" upper="4.712"/>   <!--Theoricarl limit +/-270 , Real Limit +/- 270--> 
+      <limit effort="81.5" velocity="0.42" lower="-4.702" upper="4.702"/>   <!--Theoricarl limit +/-270 , Real Limit +/- 270--> 
```

Comment:
Theoricarl -> Theoretical

I think it should be documented that we substracted 0.01 - either as a comment or by writng ```${-4.712 + 0.01}``` ?

What's the difference between theoretical and real limits? Are the theoretical limits what the motors (Schunk modules) are capable of and real what is possible in the lwa4p_extended configuration?

Regarding usage of softlimits:
What were the results of the test using the ros_control joint_limits_interfaces? Where they considered correctly there if the the safety_controller tag is activated? Any other observations with the safety_controller tag?